tests/resources/test_material_run.py

```diff
@@ -164,6 +165,30 @@ def test_filter_by_name(collection, session):
     assert sample_run['uids'] == runs[0].uids
 
 
+def test_list_by_name(collection, session):
+    all_runs = [
+        MaterialRunDataFactory(name="foo_{}".format(i)) for i in range(20)
+    ]
+    fake_request = make_fake_cursor_request_function(all_runs)
+    # lotta shady stuff going on in this test
```

Comment:
Both of these methods _do_ actually work when run against develop (in AWS) from my computer.